docs/upgrading.md

```diff
@@ -5,56 +5,16 @@ There are some major changes between the two library. This guide will help you u
 
 ## New vendor
 
-The new organization name is **PHP HTTP**. This suggests that while our main product is the HTTP Adapter project, we are not limited to it. We plan to collect any kind of HTTP related packages written in PHP.
+The new organization name is **PHP HTTP**. This suggests that while our main product is the Httplug project, we are not limited to it. We plan to collect any kind of HTTP related packages written in PHP.
```

Comment:
Don't know. I usually try to avoid including UPGRADE file in the repository, as there are no BC breaks between major releases, minor releases are usually rare and such information can go into the documentation. We should have something like this in the documentation as well, but the history and description are probably out of scope. We should mention that httplug replaces ivory, and some updating information, done.